03_scorep/scorep_notes.md

```diff
@@ -0,0 +1,69 @@
+# Score-P with BOUT++/STORM
+
+See workshop slides for intro.
+
+Score-P configuration with BOUT++ was set up with a previous project (with EPCC?). This
```

Comment:
I think it was just @d7919 in his free time :laughing: It defaults to profiling only MPI, but you can profile specific functions by including the `SCOREP0()` macro anywhere in the function.  There were also macros for timing specific regions, but I can't remember if they made it into `next`.  When we last did general profiling (c. 2018) it was through putting `SCOREP0()` almost everywhere...